nixos/modules/system/boot/systemd.nix

```diff
@@ -550,6 +550,14 @@ in
       '';
     };
 
+    systemd.unifiedCgroupHierarchy = mkOption {
```

Comment:
*bikeshed*: Can we call this `enableUnifiedCgroupHierarchy`  so it's the same naming scheme as the other boolean options in this module like `enableCgroupAccounting` ?

Reply:
bikeshed accepted and updated.